src/Symfony/Component/Form/Resources/translations/validators.lv.xlf

```diff
@@ -14,6 +14,126 @@
+                <target>Lūdzu, ievadiet korektu datumu un laiku.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Lūdzu, ievadiet korektu datumu.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Lūdzu, izvēlieties korektu failu.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Slēptā lauka vērtība ir nekorekta.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Lūdzu, ievadiet veselu skaitli.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Lūdzu, izvēlieties korektu valodu.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Lūdzu, izvēlieties korektu lokalizāciju.</target>
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please enter a valid money amount.</source>
+                <target>Lūdzu, ievadiet korektu naudas lielumu.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>Please enter a number.</source>
+                <target>Lūdzu, ievadiet skaitli.</target>
+            </trans-unit>
+            <trans-unit id="116">
+                <source>The password is invalid.</source>
+                <target>Parole ir nekorekta.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>Please enter a percentage value.</source>
+                <target>Lūdzu, ievadiet procentuālo lielumu.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>The values do not match.</source>
+                <target>Vērtības nesakrīt.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>Please enter a valid time.</source>
+                <target>Lūdzu, ievadiet korektu laiku.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>Lūdzu, izvēlieties korektu laika zonu.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>Lūdzu, ievadiet korektu tīmekļa adresi.</target>
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid search term.</source>
+                <target>Lūdzu, ievadiet korektu meklēšanas atslēgvārdu.</target>
```

Comment:
Make sure this is a "search term". ie, a sentence or a few words. 

If this translates to "keyword", doesnt that mean just one word.

Reply:
Your Latvian is suspiciously good :)

Reply:
hehe. Thank you =)